x-pack/plugins/canvas/tasks/index.js

```diff
@@ -5,14 +5,12 @@
  */
 
 import dev from './dev';
-import peg from './peg';
 import plugins from './plugins';
 import prepare from './prepare';
 import test from './test';
 
 export default function canvasTasks(gulp, gulpHelpers) {
   dev(gulp, gulpHelpers);
-  peg(gulp, gulpHelpers);
```

Comment:
is this entirely substituted by the `pegjs common/lib/grammar.peg` command on package.json?

Reply:
except for the dev part (where it watches file for changes)